src/include/distributed/distributed_planner.h

```diff
@@ -135,6 +135,24 @@ typedef struct RelationRowLock
 } RelationRowLock;
 
 
+/*
+ * Parameters to be set according to range table entries of a query.
+ */
+typedef struct RTEListProperties
+{
+	bool hasPostgresLocalTable;
+
+	bool hasReferenceTable;
+	bool hasCitusLocalTable;
+
+	/* includes hash, append and range partitioned tables */
+	bool hasDistributedTable;
+
+	/* union of above three */
```

Comment:
Minor: the above three might change in the future and this comment will probably be outdated at that point